files/SZWESAMP/ZWEGENER

```diff
@@ -0,0 +1,89 @@
+//ZWEGENER JOB                                                                  
+//*                                                                             
+//* This job is responsible for generating other jobs required                  
+//* to configure Zowe.                                                          
+//*                                                                             
+//* The method of validating your configuration is using
+//* JSON Schema <https://json-schema.org>. Zowe provides
+//* the ConfigMgr to assist in this. This job will invoke
+//* the ConfigMgr to validate your current configuration
+//* before generating any jobs. If there are any values
+//* that are incorrect, you will be notified. You should
+//* fix the value and then run this job again. You can run
+//* this job as many times as you need.
+//*
+//* Configmgr documentation:
+//* docs.zowe.org/stable/user-guide/configmgr-using
```

Comment:
Please, add `https://`
The terminal we are using in Broadcom supports opening the links with double click (which is cool), but only if the link starts with `http://` or `https://`. It will fit there:
```
----+----1----+----2----+----3----+----4----+----5----+----6----+-
//* https://docs.zowe.org/stable/user-guide/configmgr-using       
```

Reply:
good idea. yes, the terminal i uses does the same